core/src/main/scala/akka/kafka/scaladsl/Committer.scala

```diff
@@ -33,6 +33,19 @@ object Committer {
         b.commitScaladsl().map(_ => b)(ExecutionContexts.sameThreadExecutionContext)
       }
 
+  /**
+   * API may change!
+   *
+   * Batches offsets and commits them to Kafka, emits `CommittableOffsetBatch` for every batch sent for
+   * committing without expecting replies (no backpressure).
+   */
+  @ApiMayChange
+  def tellFlow(settings: CommitterSettings): Flow[CommittableOffset, CommittableOffsetBatch, NotUsed] =
```

Comment:
Yeah, that could work. I need to think about it.

Reply:
Even settings are part of the public API, right? I believe we should not bury this choice in the settings.

Reply:
That's true, but I still think it would be less surface area if it were configuration.  It also would enable us to have a less ambiguous name for it, maybe something like `commitOffsetBackpressure: Boolean`.  Otherwise, we need to add more combinations of `Committer` methods to provide users with all the different options, like I experimented with in #846.  If it's config (in `CommitterSettings`) then we can eliminate 4 potential methods (though I noticed the `Committer.sink` variant methods weren't implemented in this PR).

* `tellFlow`
* `tellFlowWithOffsetContext`
* `tellSink`
* `tellSinkWithOffsetContext`

Down the road it may also be easier to change the default settings (i.e. in 2.0) to not use backpressure than it would be to deprecate methods in `Committer`.  Though maybe I'm getting ahead of myself in that regard :)